src/app/services/user/user.service.ts

```diff
@@ -19,4 +25,42 @@ export class UserService {
       merge: true
     });
   }
+
+  getAllUsers(): Observable<UserDto[]> {
+    return this.firestoreService.getCollection(this.USER_COLLECTION)
+    .pipe(
+      map((snaps) => convertSnapshots<UserService>(snaps)),
```

Comment:
Why do you convert the result dirrectly to the service? you should do the the mapping to a different object - an object that will be used in ui